maps/CEVEris/shuttles-eris.dm

```diff
@@ -8,7 +8,7 @@
 /datum/shuttle/autodock/ferry/escape_pod/erispod/New()
 	name = "Escape Pod [number]"
 	default_docking_controller = "escape_pod_[number]"
```

Comment:
default docking controller should be the same value as arming controller, indicating the panel inside the pod

Reply:
I've spend some time trying to figure ut what does it mean.
escape_pods.dm, line 9

Reply:
Even more, pod controller and ship-side controller aren't the same controller